### ISSUE TITLE
Pages and Posts hanging when MCB is enabled

### ISSUE THREAD

**Issue by chlorinekid** [trendwerk/multiple-content-blocks#72]
I've just inherited a Wordpress site and have noticed that I can't open an item in either Pages or Posts without the site hanging. The only solution seems to be a restart of the Apache service on the server. 

There's about 20 different plugins installed and after turning everything on and then back on again one by one it seems that the issue only occurs when MCB is enabled. 

Apache error.log is recording a few errors on when it hangs:

```
zend_mm_heap corrupted
[Mon Nov 09 21:26:54.644562 2015] [core:notice] [pid 4464] AH00051: child pid 4471 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Mon Nov 09 21:26:55.645828 2015] [core:notice] [pid 4464] AH00051: child pid 4469 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Mon Nov 09 21:26:55.645966 2015] [mpm_prefork:notice] [pid 4464] AH00169: caught SIGTERM, shutting down
[Mon Nov 09 21:26:56.415475 2015] [mpm_prefork:notice] [pid 4525] AH00163: Apache/2.4.7 (Ubuntu) PHP/5.5.9-1ubuntu4.14 configured -- resuming normal operations
[Mon Nov 09 21:26:56.415550 2015] [core:notice] [pid 4525] AH00094: Command line: '/usr/sbin/apache2'

```

**Comment by haroldangenent** [comment]
That's strange. The only think I can think of is an HTTP request firing to the front-end, with a slow front-end.

Does this also happen if you disable HTTP requests in Settings > Multiple content blocks?

**Comment by chlorinekid** [comment]
Mmm for some reason I don't appear to have MCB listed in Settings. Why might that be?

**Comment by haroldangenent** [comment]
What version are you running?

**Comment by chlorinekid** [comment]
I'm running v3.2.2, latest Wordpress, PHP 5.5.9 on Ubuntu 14.04LTS

I've managed to get into Pages and Post items by disabling HTTP requests in Settings.

MCB wasn't showing up in Settings initially so I removed the plugin completely and reinstalled it. It then showed up in Settings. I had a quick peek but left HTTP requests ON and went back to Pages to see if it still crashed after a fresh install. It did. I then went back to Settings to find that MCB had disappeared again :)

So...

Removed it again, reinstalled, went to Settings and disabled HTTP requests. All working for the time being but certainly an issue somewhere down the line. Hope that helps.
